Mayhemfile

```diff
@@ -0,0 +1,6 @@
+project: lwgps
```

Comment:
I recommend renaming your target to clarify what is being fuzzed.

Reply:
I've renamed it to fuzz_nmea_parser